back/test/integration_tests/test_permissions.py

```diff
@@ -97,15 +97,15 @@
         [
             "another_dev_coordinatorr@boxcare.org",
             BOX_CARE_ID,
-            [THESSALONIKI_ID],
-            [SAMOS_ID, ATHENS_ID],
+            [ATHENS_ID],
```

Comment:
@pylipp I just realized that in other PRs these tests fail, too. This fix should solve it. 
The problem is that another_dev_coordinatorr and another_dev_volunteer are associated with base Athens in the db seed, but with base Thessaloniki in Auth0. 
I fixed this and synced it.